### PR TITLE
Adding epmin and epmax conditional control attributes

### PR DESCRIPTION
**Pull request by asoloway64**

### CODE REVIEW

File: draft-ietf-core-dynlink.md [hunk 1]
```diff
@@ -106,6 +105,7 @@ Conditional Control Attributes are defined below:
 | Attribute         | Parameter | Value            |
 | Minimum Period (s)| pmin      | xs:decimal (>0) |
 | Maximum Period (s)| pmax      | xs:decimal (>0) |
+| Change Step       | st        | xs:decimal (>0) |
```
Comment (bsilverajan) [inline draft-ietf-core-dynlink.md]: I added change step to this table to keep it consistent with its usage together with pmin and pmax. Does this sound good?

File: draft-ietf-core-dynlink.md [hunk 2]
```diff
@@ -106,6 +105,7 @@ Conditional Control Attributes are defined below:
...
 | Minimum Evaluation Period (s)| epmin      | xs:decimal (>0) |
 | Maximum Evaluation Period (s)| epmax      | xs:decimal (>0) |
 {: #weblinkattributes title="Conditional Control Attributes"}
```
Comment (bsilverajan) [inline draft-ietf-core-dynlink.md]: The Travis CI build is failing on this document owing to "weblinkattributes" being a duplicate label from the previous table. I think it's better to rename "weblinkattributes" on this line to, for example, "controlattributes".

File: draft-ietf-core-dynlink.md (hunk 1 above)
Comment (asoloway64) [reply draft-ietf-core-dynlink.md]: Should 'st' be in the Notification Attributes table?

File: draft-ietf-core-dynlink.md (hunk 2 above)
Comment (asoloway64) [reply draft-ietf-core-dynlink.md]: Should we change #weblinkattributes to #notificationattributes in the Notification Attributes as well?

Comment (asoloway64) [reply draft-ietf-core-dynlink.md]: I can't find any documentation on this title format and it only displays as a table entry for me. I am using the browser for github access.

**Pull request merged**